services/frontend/src/components/skillsCard/SkillsView.jsx

```diff
@@ -2,34 +2,31 @@ import PropTypes from "prop-types";
 import { Tag, Empty, Row, Col } from "antd";
 import { TagTwoTone } from "@ant-design/icons";
 import { FormattedMessage } from "react-intl";
+import { Fragment } from "react";
 
 const SkillsView = ({ categoriesSkills, skills }) => {
   if (skills.length > 0)
-    return (
-      <>
-        {categoriesSkills.map(
-          (categorySkill) =>
-            categorySkill != null && (
-              <>
-                <Row align="middle">
-                  <Col>
-                    <TagTwoTone twoToneColor="#3CBAB3" />
-                  </Col>
-                  <Col>
-                    <FormattedMessage id={categorySkill.val} />:
-                  </Col>
-                </Row>
-                <Row>
-                  <Col>
-                    {skills[categorySkill.index].val.map((skill) => (
-                      <Tag color="#00605e">{skill}</Tag>
-                    ))}
-                  </Col>
-                </Row>
-              </>
-            )
-        )}
-      </>
+    return categoriesSkills.map(
```

Comment:
you removed the empty fragment if the map is empty. are sure this isn't going to cause issue

Reply:
It's only runs if there is a skill and if there is a skill, it must belong to a category, no?

Reply:
if it doesn't effect the behavior then im ok with it